services/frontend/www-app/index.html

```diff
@@ -1,6 +1,69 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <script type="text/javascript">
+      // The assets produced by our build process have content specific URLs like index.123456.js
+      // which gets baked into index.html.
+      //
+      // However, it's possible that the user's browser will cache this index.html, referencing old assets
+      // that are no longer deployed.
+      // This script will detect if the index.CACHE_BUSTER.js referenced by this file is stale and reload
+      // the page if so.
+      //
+      // Our build process inserts the index.CACHE_BUSTER.js script tag at the end of the head tag, so
+      // this script needs to execute before that.
+      const headNode = document.querySelector('head');
```

Comment:
This seems pretty janky, but it's the least worst idea I had for a quick fix.

Reply:
I do agree that this is janky, but if it would let us keep index.html caching it might be worth it. It's not like we run a particularly low-latency service, especially west of the atlantic.

Reply:
If I've got this javascript right, it should only result in a slowdown for a page that is broken. Slow seems better than broken for sure. And it shouldn't measurably affect the happy path.